newsfragments/2774.doc.rst

```diff
@@ -0,0 +1 @@
+Changed the names of `m` and `n` parameters to `threshold` and `num_kfrags`.
```

Comment:
I think either we add an additional `removal` newsfragment or this might need to be changed to a `removal` newsfragment, since we are deprecating/modifying public-facing APIs that is potentially currently used:
1.  Character control: "m" and "n" parameters can no longer be used
2. CLI: "--m" and "--n" are no longer valid -> "-m" and "-n"
3. Python API: no longer takes "m" and "n" as parameter.

All in favour of `threshold` and `num_kfrags'